### DISCUSSION TITLE
Dynamic Torch

### DISCUSSION THREAD

**Discussion by ghost** [jebbyk/OSBES-minecraft-bedrock-edition-shader#192] [Ideas]
Hello there,

A little feature which could be awesome : adding dynamic torch. While holding a light source in right hand or left hand(aka a torch, or a lantern...), it will automatically produces light, as of a more realistic way of the light systems. 
Also, I see out there that they are addons which allow the player to hold a torch in left hand. If this feature can be added alongside the dynamic torch it would be epic !
Thanks for reading :)

See link to see what I mean (english isn't my first language so tell me if you don't understand what I said).
[https://youtu.be/_d0PhGLTu-o](Dynamic Torch Example)

**Comment by LanceAldrei04** [comment]
Just download Dynamic lighting lake that in mcpedl.com or any store.

**Reply by jebbyk** [reply]
are you sure it will not break textures?
